e2e/README.md

```diff
@@ -2,19 +2,28 @@
 
 Tooling and utilities for E2E testing of `radicle-link` networks.
 
-## Lo-fi Local Devnet
+## Prequisites
 
-Prequisites:
+* [podman] or [docker]
+* [docker-compose]
+* Optional: [overmind]
 
-    * [overmind](https://github.com/DarthSim/overmind)
-    * [podman](https://podman.io)
+## Usage
 
-Run [`./localnet`](./localnet) to compile a simple peer against `HEAD`, and run
-it. Once it started up, you can watch some (rudimentary) metrics via the 'theus
-web interface at [http://localhost:9090](http://localhost:9090).
+To quickly spin up a few peers without waiting for slow container builds, run
+the `quick` script. This will start `$NUM_PEERS`, plus a bootstrap node, using
+[overmind].
 
-You can adjust the number of peers by setting the `NUM_PEERS` environment
-variable.
+A more elaborate setup is provided via [docker-compose]. Run
 
-_Note that this is currently simply a debugging aid. More sophisticated
-orchestration may be added in the future (contributions welcome!)_
+    DOCKER_BUILDKIT=1 docker-compose up
```

Comment:
When trying to run this I got:
```
ERROR: 
        Can't find a suitable configuration file in this directory or any
        parent. Are you in the right directory?

        Supported filenames: docker-compose.yml, docker-compose.yaml
```

Renaming `compose.yaml` to `docker-compose.yaml` allowed it to work. Maybe a versioning issue?

Reply:
You should be able to tell it which yamls to look at using `docker-compose -f the.yaml`. I'd rather change the readme than the filename, because it's more likely that we will have more than one compose file.

Reply:
Cool, good to know :+1: